com.unity.render-pipelines.universal/Runtime/ForwardRenderer.cs

```diff
@@ -73,6 +93,10 @@ public ForwardRenderer(ForwardRendererData data) : base(data)
             m_DefaultStencilState.SetFailOperation(stencilData.failOperation);
             m_DefaultStencilState.SetZFailOperation(stencilData.zFailOperation);
 
+            m_ForwardLights = new ForwardLights();
```

Comment:
do we need to create forward lights if the asset is set to deferred?

Reply:
Yes, for the forward only pass.